process/transaction/shardProcess.go

```diff
@@ -543,6 +557,31 @@ func (txProc *txProcessor) processMoveBalance(
 	return nil
 }
 
+func (txProc *txProcessor) removeConsumedValueFromSender(
```

Comment:
`refund` instead of `remove`?

Reply:
updated to revertConsumedValueFromSender